Example/IntegrationTests/Stubs/SignerFactory.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import CryptoSwift
+import Web3
+import Auth
+
+public struct AuthSignerFactory: SignerFactory {
```

Comment:
could we have same sample app utils to not duplicate this class?

[Example/IntegrationTests/Stubs/SignerFactory.swift](https://github.com/WalletConnect/WalletConnectSwiftV2/pull/607/files#diff-3f4081f5d5a85b15859f7954ec96f8445dbc55a42065224c886174edef249ccc)

[Example/Showcase/Classes/DomainLayer/SignerFactory/AuthSignerFactory.swift](https://github.com/WalletConnect/WalletConnectSwiftV2/pull/607/files#diff-5ded2a137e4f9fb72d3b20758450de5960ce9ed0b314ee3265894030320f7987)

Reply:
Created Shared folder with DefaultSocketFactory and DefaultSignerFactory which is shared between Wallet, DApp, Showcase and IntegrationTests targets